main/src/com/google/refine/clustering/binning/FingerprintKeyer.java

```diff
@@ -85,6 +85,8 @@
             .put("œ", "oe")
             .put("ẜ", "s") // more long S forms
             .put("ẝ", "s")
+            .put("Ƿ","w") // Wynn from Old English modernly transliterated to w
```

Comment:
Is case folding not working correctly? We shouldn't need two entries.

As long as we're doing this, should we map the rest of the wynn forms? http://www.fileformat.info/info/unicode/char/search.htm?q=wynn&preview=entity
All runes? http://www.fileformat.info/info/unicode/char/search.htm?q=runic&preview=entity

Reply:
You can take this PR over Tom.  I've got what I needed out of of it unblock me.  If it's useful for others or needs more work to be more useful...have at it.

Reply:
The mapping that I was following was from the Unicode spec itself.  In particular the `scf` and `slc`, which I still find a bit weird on some glyphs.

Reply:
and where I was trying to get `fingerprint` and `ngramFingerprint` to follow the rules of `scf` and `slc` by default.  If that's reading too much into it, then I goofed and sorry, but felt they were sensible most of the time for Old English texts.

Reply:
The string is lowercased before it gets to this point, so only the lowercase substitution is needed.